### ISSUE TITLE
Show recommended color sets for popular colorschemes

### ISSUE THREAD

**Issue by DarrienG** [jaxbot/semantic-highlight.vim#76]
It would be nice if there were recommended color sets for recommended colorschemes.

Have a doc recommended.md or the like and have something like:

- solarized light
- solarized dark
- gruvbox
- etc.

**Comment by shangxiao** [comment]
Yep agreed, I have a dark background which I can't change.  It'd be super handy to have an example colour set example that I can just throw at it so I can evaluate the plugin.

**Comment by 3ddiddE** [comment]
Is there any reference/website at all that has a working set of colors for a dark scheme?
Or are there any updates on this?

**Comment by joetsoi** [comment]
I did a search of "semanticGUIColors" across github and came across someone using https://medialab.github.io/iwanthue/ which seems like an easy way to generate a set that you can test.
